flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -562,6 +562,13 @@ abstract class TableTestUtil(
     testingTableEnv.scan(name)
   }
 
+  /**
+   * Registers a [[UserDefinedFunction]] according to FLIP-65.
+   */
+  def addTemporarySystemFunction(name: String, function: UserDefinedFunction): Unit = {
```

Comment:
Why is this added? It's not used anywhere?

Reply:
Good point, I will add usage. It is definitely necessary to have in the utils otherwise we have no means to test new functions.